.github/workflows/triage-labelled.yml

```diff
@@ -93,6 +93,9 @@ jobs:
     runs-on: ubuntu-latest
     if: >
       contains(github.event.issue.labels.*.name, 'A-New-Search-Experience') ||
+      (contains(github.event.issue.labels.*.name, 'A-Threads')
+        && (contains(github.event.issue.labels.*.name, 'S-Major') ||
```

Comment:
I prefer leaving the `&&` at the end of the previous line so it's stylistically similar to where I would expect the `||` and matches more closely with what other projects do